src/Core/Bootstrap/ConfigurationLoader.php

```diff
@@ -79,6 +74,30 @@ protected function loadConfigurationFiles(Application $app, RepositoryContract $
         }
     }
 
+    /**
+     * Get the path to the routes cache file.
+     *
+     * @return string
+     */
+    protected function maybeForceWpConfigInclude()
+    {
+        // Avoid duplicate constants definitions.
+        if ($this->isWordPressConfigAlreadyLoaded()) {
+            return;
+        }
+        $cacheConfig = app()->getCachedConfigPath('config.php');
+        file_exists($cacheConfig) ? require_once app()->configPath('wordpress.php') : null;
```

Comment:
As we don't return anything here, I suggest to update the method body like so:
```suggestion
        if (! file_exists($cacheConfig)) {
            return;
        }
        
        require_once app()->configPath('wordpress.php');
```